packages/core/src/__integrationtests__/AccountLinking.spec.ts

```diff
@@ -180,11 +180,12 @@ describe('When there is an on-chain DID', () => {
       }, 40_000)
 
       it('should be possible to associate the account while the sender pays the deposit', async () => {
-        const linkAuthorisation = await AccountLinks.authorizeLinkWithAccount(
-          keypair.address,
-          did.identifier,
-          signingCallback
-        )
+        const linkAuthorisation =
+          await AccountLinks.getAuthorizeLinkWithAccountTx(
```

Comment:
Do we follow British or American English in the SDK? Before it was British (e.g. _Authori**s**ation_), now it's American (_authori**z**e_).

In the mashnet-node, we apply American as we follow Substrate.

Reply:
@Dudleyneedham tries to sneak in british english, but I would prefer US